tools/src/main/java/com/emc/ecs/nfsclient/nfs/io/FileReadingTest.java

```diff
@@ -0,0 +1,101 @@
+/**
```

Comment:
Seems like you have multiple "tools" now, so they need to be in separate subdirectories to follow the common-build conventions.  Sorry if these aren't abundantly clear, but each jar file must have a single main class, which is built automatically... so if you want this to be a little test utility, put it in its own subdir with its own build file and "include" it in settings.gradle